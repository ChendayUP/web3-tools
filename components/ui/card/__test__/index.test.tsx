import { render, screen } from "@testing-library/react"

import { Card } from "../index"

describe("Back", () => {
  test("should render with default text", () => {
    render(
      <Card>
        <p>card</p>
      </Card>
    )

    const element = screen.getByText(/card/i)
    screen.debug(element)
    expect(element).toBeInTheDocument()
  })

  test("should render <p>card</p>", () => {
    const { asFragment } = render(
      <Card>
        <p>card</p>
      </Card>
    )
    expect(asFragment()).toMatchSnapshot()
  })

  it("should render <Card/>", () => {
    const { asFragment } = render(<Card />)

    expect(asFragment()).toMatchSnapshot()
  })
})
