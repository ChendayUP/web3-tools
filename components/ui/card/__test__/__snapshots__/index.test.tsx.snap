// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Back should render <Card/> 1`] = `
<DocumentFragment>
  <div
    class="rounded-xl border bg-card text-card-foreground shadow"
  />
</DocumentFragment>
`;

exports[`Back should render <p>card</p> 1`] = `
<DocumentFragment>
  <div
    class="rounded-xl border bg-card text-card-foreground shadow"
  >
    <p>
      card
    </p>
  </div>
</DocumentFragment>
`;
